trade/trader_fameex.js

```diff
@@ -0,0 +1,856 @@
+          type: formatOrderType(order.orderType),
+          timestamp: order.createTime,
+          amount: +order.money,
+          amountExecuted: +order.filledAmount,
+          amountLeft: +order.filledAmount - +order.money,
+          status: formatOrderStatus(order.state),
+        }));
+
+        return result;
+      } catch (error) {
+        log.warn(`Error while processing getOpenOrders(${paramString}) request results: ${JSON.stringify(ordersData)}. ${error}`);
+        return undefined;
+      }
+    },
+
+    /**
+     * List of all account open orders
+     * !POSSIBLE IMPLEMENTATION ERRORS!
+     * !At the moment it is impossible to implement this functional correctly, due to problems on the FameEX side
+     * @param {String} pair In classic format as BTC/USDT
+     * @returns {Promise<Array|undefined>}
+     */
+    async getOpenOrders(pair) {
+      const allOrders = [];
+      const coinPair = formatPairName(pair);
+
+      let ordersInfo;
+      let pageNum = 1;
+
+      do {
+        ordersInfo = await this.getOpenOrdersPage(coinPair, pageNum);
+
+        if (!ordersInfo) return undefined;
+
+        allOrders.push(...ordersInfo);
+
+        pageNum += 1;
+      } while (ordersInfo.length === ordersMaxPageSize);
+
+      return allOrders;
+    },
+
+    /**
+     * Get specific order details
+     * What's important is to understand the order was filled or closed by other reason
+     * status: unknown, new, filled, part_filled, cancelled
+     * @param {String} orderId Example: '10918742125338689536'
+     * @param {String} pair In classic format as BTC/USDT
+     * @returns {Promise<Object|undefined>}
+     */
+    async getOrderDetails(orderId, pair) {
+      const paramString = `orderId: ${orderId}, pair: ${pair}`;
+      const coinPair = formatPairName(pair);
+
+      let order;
+
+      try {
+        order = await fameEXApiClient.getOrderDetails(coinPair.pairDash, orderId);
+      } catch (error) {
+        log.warn(`API request getOrderDetails(${paramString}) of ${utils.getModuleName(module.id)} module failed. ${error}`);
+        return undefined;
+      }
+
+      try {
+        if (order.code === successCode) {
+          return {
+            orderId: order.data.orderId,
+            tradesCount: undefined, // FameEX doesn't provide trades
+            price: +order.data.price,
+            side: order.data.side === orderSides.buy ? 'buy' : 'sell',
+            type: formatOrderType(order.data.orderType),
+            amount: +order.data.money,
+            volume: +order.data.money * +order.data.triggerPrice,
+            pairPlain: coinPair.pairPlain,
+            pairReadable: coinPair.pairReadable,
+            totalFeeInCoin2: +order.data.filledFee,
+            amountExecuted: +order.data.filledAmount,
+            volumeExecuted: +order.data.filledMoney,
+            timestamp: order.data.createTime,
+            updateTimestamp: order.data.updateTime,
```

Comment:
`createTime: 1694176590`

Whilst we need in format: `1 694 176 590 000`

```suggestion
            timestamp: order.data.createTime * 1000,
            updateTimestamp: order.data.updateTime * 1000,
```

Reply:
fixed